code/__DEFINES/logging.dm

```diff
@@ -7,6 +7,7 @@
 #define INVESTIGATE_RECORDS			"records"
 #define INVESTIGATE_SINGULO			"singulo"
 #define INVESTIGATE_SUPERMATTER		"supermatter"
+#define INVESTIGATE_ANTIMATTER		"antimatter"
```

Comment:
Just put it in supermatter log and rename it tbh

Reply:
Yeah, was thinking about that.